app/views/organisations/edit.html.erb

```diff
@@ -0,0 +1,40 @@
+<% content_for :title, "Change #{@organisation.name}'s details" %>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link(
+    text: 'Back',
+    href: :back,
+  ) %>
+<% end %>
+
+<%= form_for(@organisation, as: :organisation, html: { method: :patch }) do |f| %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">
+        <%= content_for(:title) %>
+      </h1>
+
+      <%= f.govuk_text_field :name,
+        autocomplete: "name"
+      %>
+
+      <%= f.govuk_text_field :address_line1,
+        autocomplete: "address-line1"
```

Comment:
I haven’t tested this, but something like:

```suggestion
        label: -> { safe_join(["Building and street", " ", tag.span("line 1 of 2", class: "govuk-visually-hidden")]) },
        autocomplete: "address-line1"
```

Output should be to display a label with visible text ‘Building and street’ and hidden text ‘ line 1 of 2’.